next.json

```diff
@@ -414,9 +438,84 @@
             }
         ]
     },
+    "depth stencil state": {
+        "category": "object"
+    },
+    "depth stencil state builder": {
+        "category": "object",
+        "methods": [
+            {
+                "name": "get result",
+                "returns": "depth stencil state"
+            },
+            {
+                "name": "set depth enabled",
```

Comment:
I think this is a good idea. Do we want to do something like this for the stencil test as well? We could also infer that the stencil test will do nothing based on the stencil operation (stencilFail: Keep, depthFail: Keep, depthPass: Replace).

However for the stencil test it might be annoying to remember what values disable it so perhaps a helper function in the API to enable/disable would still be useful?

Or I suppose the stencil would default to `stencilFail: Keep, depthFail: Keep, depthPass: Replace` (off). And any changes to that would imply you wanted to enable the stencil test?